src/content/docs/tutorial-customer-experience/intro-to-customer-experience.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Improve your customers' digital experience with New Relic" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+  - /docs/new-relic-solutions/observability-maturity/qf-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/quality-foundation-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/customer-experience/customer-experience-quality-foundation-guide
+---
+
+import cxWhatYouCanMeasureNr from 'images/cx_diagram_what-you-can-measure-nr.webp'
+
+Digital customer experience refers to the quality of how your users interact with your service across all your digital touch points (web, mobile, and IoT). Because it has so many moving parts within the concept, it can be difficult to digest when taken as whole. You need to manage the availability of your service, its performance to ensure that it's usable, the quality of the content that gives users the information they need, and the relevance of your products and content so that users remain engaged.
```

Comment:
I don't think we need to get into the difficulty. I assume if someone's here, they're already looking for some help. 

I'd delete those first two sentences and focus on that last sentence. 

The key idea here is, I think: There's all these things that make up a high-quality digital experience. New Relic can help you make sure that's happening.

Reply:
One of the things I think we need to focus on for tutorials is in making sure we define concepts from the beginning so that we ensure the reader is on the same page as we are, but I definitely agree with removing the mention of difficulty so I have edited that out. Thanks for the suggestion!

Reply:
Thanks for the extra context :)